app/models/service_template.rb

```diff
@@ -10,6 +10,20 @@ class ServiceTemplate < ApplicationRecord
     "storage"         => _("Storage")
   }.freeze
 
+
+  CATALOG_ITEM_TYPES = {
```

Comment:
@martinpovolny addressed you comment, moving code to display nice value for prov_type back to UI PR.